lib/commands/sync.js

```diff
@@ -192,12 +202,36 @@ module.exports = function *(argv) {
     let queue = blockingQueue()
 
     console.log(`Synchronizing ${totalRecords} records in ${table}...                                                                        `)
-    let sourceCursor = yield sr.db(sourceDB).table(table).orderBy({index: sr.asc('id')})
-      .map(function (row) { return {id: row('id'), hash: sr.uuid(row.toJSON())} })
-      .run({cursor: true})
-    let targetCursor = yield tr.db(targetDB).table(table).orderBy({index: tr.asc('id')})
-      .map(function (row) { return {id: row('id'), hash: tr.uuid(row.toJSON())} })
-      .run({cursor: true})
+    let sourceCursor
+    let targetCursor
+    if (companyIds) {
+      console.log(`Synchronizing by Companies: ${companyIds}`)
+      sourceCursor = yield sr.db(sourceDB).table(table)
+        .orderBy({index: "id"})
+        .filter(el =>
+          sr.expr(companyIds)
+          .contains(el("Company"))
+        )
```

Comment:
Change "el" to something more descriptive (e.g. "doc")

Reply:
changed to doc